lib/src/model/documentation_comment.dart

```diff
@@ -72,11 +65,11 @@ mixin DocumentationComment
   /// like `///`, `//`, `/*`, `*/`.
   String get documentationComment => element.documentationComment ?? '';
 
-  /// True if [this] has a synthetic/inherited or local documentation
-  /// comment.  False otherwise.
+  /// Whether `this` has a synthetic/inherited or local documentation comment,
+  /// and false otherwise.
   bool get hasDocumentationComment => element.documentationComment != null;
 
-  /// Returns true if the raw documentation comment has a 'nodoc' indication.
+  /// Whether the raw documentation comment has a 'nodoc' indication.
```

Comment:
Instead of saying "has a 'nodoc' indication", can we just write out what 'nodoc' means and what it does? Otherwise, someone reading it would have to search to find what 'nodoc' is anyways.

Reply:
Yeah that is weird verbiage. Done!